index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ERNEST Portfolio Website</title>
+</head>
+<body>
+    <!-- defining the header and wraping it in divs and will contain the logo -->
+    <div class="main_container">
+        <div class="inside_container">
+            <header class="main_header">
+              <nav class="navbar">
+                <a href="#" class="my-portfolio-logo">
+                    <img src="./assets/MyLogo.png" alt="My logo"/>
+                </a>
+                <a>
+                    <img src="./assets/menu.png" alt="menu-icon"/>
+                </a>
+              </nav>
+            </header>
+    <!-- Defining the main section for the description of the developer -->
+            <main class="main_section">
+                <div class="container">
+                   <div class="welcome_note"> <h2 class="h2">I am Ernest Lasten. Glad to see you!</h2></div>
```

Comment:
- [ ] Kindly make sure to put the title in an `h1` tag instead of `h2`. The H1 tag is the one that comes first and shows search engines what content can be expected on the rest of the page.